From 5aefa0d8ad9737123e3f3580d8c5d0ae560a5891 Mon Sep 17 00:00:00 2001
From: Ben Whitten <ben.whitten@lairdtech.com>
Date: Thu, 20 Apr 2017 12:04:19 +0100
Subject: [PATCH 1/3] laird: Remove TX power check with LUT

It is the job of the HAL to check transmit powers, the packet will be
transmitted at a lower power then requested if an LUT entry does not
exist.

Bug 10976
---
 lora_pkt_fwd/src/lora_pkt_fwd.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/lora_pkt_fwd/src/lora_pkt_fwd.c b/lora_pkt_fwd/src/lora_pkt_fwd.c
index 801f28d..ae94c84 100644
--- a/lora_pkt_fwd/src/lora_pkt_fwd.c
+++ b/lora_pkt_fwd/src/lora_pkt_fwd.c
@@ -2522,8 +2522,7 @@ void thread_down(void) {
                 }
                 if (i == txlut.size) {
                     /* this RF power is not supported */
-                    jit_result = JIT_ERROR_TX_POWER;
-                    MSG("ERROR: Packet REJECTED, unsupported RF power for TX - %d\n", txpkt.rf_power);
+                    MSG("WARNING: Unsupported RF power for TX - %d, will transmit at lower power\n", txpkt.rf_power);
                 }
             }
 
-- 
2.17.1

